src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -40,6 +40,21 @@
     end
   end
 
+  describe '#word_break' do
+    it "continuously adds tag after N characters" do
+      expect(word_break("XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", 10)).to \
+        eq("XXXXXXXXXX<wbr>XXXXXXXXXX<wbr>XXXXXXXXXX<wbr>XXXXXXXXXX")
+    end
+
+    it "adds no tag if string is shorter than N characters" do
+      expect(word_break("XX", 10)).to eq("XX")
```

Comment:
Not really sure if I got you right. Since the test tests if NO tag was added if the given amount of characters is shorter than the second parameter.